### ISSUE TITLE
npm released code not match current source

### ISSUE THREAD

**Issue by katopz** [pancakeswap/pancake-swap-sdk#32]
in released `npm` (bad)
```json
  "dependencies": {
    "@pancakeswap-libs/pancake-swap-core": "^0.1.0",
    "big.js": "^5.2.2",
    "decimal.js-light": "^2.5.0",
    "jsbi": "^3.1.1",
    "tiny-invariant": "^1.1.0",
    "tiny-warning": "^1.0.3",
    "toformat": "^2.0.0"
  }
```

current source (good)
```json
  "dependencies": {
    "big.js": "^5.2.2",
    "decimal.js-light": "^2.5.0",
    "jsbi": "^3.1.4",
    "tiny-invariant": "^1.1.0",
    "tiny-warning": "^1.0.3",
    "toformat": "^2.0.0"
  },
```

and `"@pancakeswap-libs/pancake-swap-core": "^0.1.0",` currently depend to `@pancakeswap-libs/pancake-swap-core`
which has `truffle` that not compatible with node 16 and lead to build failed apparently.

## TLDR
Please release current source and bump version.

**Comment by Chef-Cheems** [comment]
Hi! If you mean 2.4.1 and 2.4.2 versions - those were "broken" (forgot to run build before releasing).
Version 2.4.3 in what you see right now in the repo. We removed `pancake-swap-core` dependency cause it was used just for 1 ABI which now lives inside this repo.
Please upgrade to 2.4.3 🙂

**Comment by katopz** [comment]
> Hi! If you mean 2.4.1 and 2.4.2 versions - those were "broken" (forgot to run build before releasing). Version 2.4.3 in what you see right now in the repo. We removed `pancake-swap-core` dependency cause it was used just for 1 ABI which now lives inside this repo. Please upgrade to 2.4.3 🙂

Thanks, In that case you better use github action for auto build and release next time. 😅
